### PR TITLE
Update page for Docker Desktop for Linux tech preview

### PR DESCRIPTION
**Pull request by aiordache**
Update page with link to the tech preview build and current known issues.

### CODE REVIEW

File: desktop/linux/index.md [hunk 1]
```diff
@@ -124,7 +137,10 @@ $ systemctl --user stop docker-desktop
 
 ## Logs
 
-If you experience any issues, you can access Docker Desktop logs by running the following command:
+To create and upload a diagnostics bundle select
```
Comment (usha-mandya) [inline desktop/linux/index.md:140]:
```suggestion
To create and upload a diagnostics bundle, from the Docker menu, select
```

Comment (usha-mandya) [inline desktop/linux/index.md:140]:
```suggestion
To create and upload a diagnostics bundle:

1. From the Docker menu, select **Troubleshoot** > **Get support**.
2. When the diagnostics are available, click **Upload to get a Diagnostic ID**.
3. Make a note of the Diagnostic ID displayed on the Support page. You can send this ID with your bug report to investigate any issues.
```

File: desktop/linux/index.md [hunk 2]
```diff
@@ -144,7 +160,12 @@ $ sudo apt remove docker-desktop
 
 ## Known issues
 
- - The **Reset to factory defaults** option on the **Troubleshoot** page currently does not work.
+ - Docker CLI login flow presents some inconsistencies that are under investigation. If problems are encountered when trying to log in, remove the `credsStore` property from `~/.docker/config.json` and restart Docker Desktop (run either
```
Comment (usha-mandya) [inline desktop/linux/index.md:163]:
```suggestion
 - The Docker CLI login flow has some inconsistencies that we are currently investigation. If you experience any issues when trying to log in, remove the `credsStore` property from `~/.docker/config.json` and restart Docker Desktop (run either 
```

File: desktop/linux/index.md [hunk 3]
```diff
@@ -144,7 +160,12 @@ $ sudo apt remove docker-desktop
 
 ## Known issues
 
- - The **Reset to factory defaults** option on the **Troubleshoot** page currently does not work.
+ - Docker CLI login flow presents some inconsistencies that are under investigation. If problems are encountered when trying to log in, remove the `credsStore` property from `~/.docker/config.json` and restart Docker Desktop (run either 
+ `systemctl --user restart docker-desktop` or quit Docker Desktop and relaunch).
+   * Docker Desktop stores the passwords in base-64 encoded plaintext, integration with `pass` is work in progress.
```
Comment (usha-mandya) [inline desktop/linux/index.md:165]:
```suggestion
  - Docker Desktop stores the passwords in base-64 encoded plaintext, integration with `pass` is work in progress.
```

File: desktop/linux/index.md [hunk 4]
```diff
@@ -144,7 +160,12 @@ $ sudo apt remove docker-desktop
...
+
+ - Integration with the scan plugin is still work in progress.
```
Comment (usha-mandya) [inline desktop/linux/index.md:167]:
```suggestion
 - Integration with the `docker scan` plugin is still a work in progress. 
```

File: desktop/linux/index.md [hunk 5]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
 2. Download and install the Tech Preview Debian package:
     ```console
-    $ curl https://desktop-stage.docker.com/linux/main/amd64/73772/docker-desktop.deb --output docker-desktop.deb
+    $ curl https://desktop-stage.docker.com/linux/main/amd64/74134/docker-desktop.deb --output docker-desktop.deb
     $ sudo apt install ./docker-desktop.deb
     ```
-3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration.
+3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration (automated in post-install script). 
+
+There are a few post-install configurations done via the maintainers' scripts (postinstall script contained in the deb package):
```
Comment (usha-mandya) [inline desktop/linux/index.md:32]:
```suggestion
There are a few post-install configuration steps done through the maintainers' scripts (post-install script contained in the deb package):
```

File: desktop/linux/index.md [hunk 6]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+- for each user we do:
```
Comment (usha-mandya) [inline desktop/linux/index.md:33]:
```suggestion
- For each user, the post-install script:
```

File: desktop/linux/index.md [hunk 7]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+    - install systemd units
```
Comment (usha-mandya) [inline desktop/linux/index.md:34]:
```suggestion
    - installs systemd units
```

File: desktop/linux/index.md [hunk 8]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+    - configure `desktop-linux` as the default docker cli context
```
Comment (usha-mandya) [inline desktop/linux/index.md:35]:
```suggestion
    - configures `desktop-linux` as the default Docker CLI context
```

File: desktop/linux/index.md [hunk 9]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+    - install compose and scan plugins to `~/.docker/cli-plugins`
```
Comment (usha-mandya) [inline desktop/linux/index.md:36]:
```suggestion
    - installs Compose and the `docker scan` plugins to `~/.docker/cli-plugins`
```

File: desktop/linux/index.md [hunk 10]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+    - enable compose V2 as the default docker-compose
```
Comment (usha-mandya) [inline desktop/linux/index.md:37]:
```suggestion
    - enables Compose V2 as the default `docker-compose`
```

File: desktop/linux/index.md [hunk 11]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+    - add user to `docker` and `kvm` group
```
Comment (usha-mandya) [inline desktop/linux/index.md:38]:
```suggestion
    - adds user to `docker` and `kvm` groups
```

File: desktop/linux/index.md [hunk 12]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+
+- set capability on the Docker Desktop binary to be allowed to map privileged ports and set resource limits
```
Comment (usha-mandya) [inline desktop/linux/index.md:40]:
```suggestion
- sets the capability on the Docker Desktop binary to map privileged ports and set resource limits
```

File: desktop/linux/index.md [hunk 13]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+- add a dns name for kubernetes to `/etc/hosts`
```
Comment (usha-mandya) [inline desktop/linux/index.md:41]:
```suggestion
- adds a DNS name for Kubernetes to `/etc/hosts`
```

File: desktop/linux/index.md [hunk 14]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
...
+- create the desktop file for the applciation launcher
```
Comment (usha-mandya) [inline desktop/linux/index.md:42]:
```suggestion
- creates the Docker Desktop file for the application launcher
```

File: desktop/linux/index.md [hunk 15]
```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
 1. Set up the [Docker repository](../../engine/install/ubuntu.md#install-using-the-repository).
 2. Download and install the Tech Preview Debian package:
     ```console
-    $ curl https://desktop-stage.docker.com/linux/main/amd64/73772/docker-desktop.deb --output docker-desktop.deb
+    $ curl https://desktop-stage.docker.com/linux/main/amd64/74134/docker-desktop.deb --output docker-desktop.deb
     $ sudo apt install ./docker-desktop.deb
     ```
-3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration.
+3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration (automated in post-install script).
```
Comment (fredericdalleau) [inline desktop/linux/index.md:30]: Is there no way for the user to just reload his groups ?
Traditionnally `su $(whoami)` should open a shell with the new groups available.

File: desktop/linux/index.md (hunk 3 above)
Comment (fredericdalleau) [inline desktop/linux/index.md:165]: What happens after the release, will we store all the password in pass, and remove that in the credStore?

File: desktop/linux/index.md (hunk 2 above)
Comment (ebriney) [reply desktop/linux/index.md]: If you remove `credStore` property, you will need to close the dashboard and reopen it to get in sync with the cli state.

File: desktop/linux/index.md (hunk 15 above)
Comment (aiordache) [reply desktop/linux/index.md:30]: We are already doing a daemon reload in the post-install script. We wanted just to have users check that worked.

File: desktop/linux/index.md (hunk 3 above)
Comment (ebriney) [reply desktop/linux/index.md]: WDYM @fredericdalleau? `pass` is the credStore

Comment (aiordache) [reply desktop/linux/index.md]: Yes, this is the current plan.

Comment (ebriney) [reply desktop/linux/index.md]: Plaintext is saved as base64 "user:pass" in config.json when we don't use any credStore

Comment (fredericdalleau) [reply desktop/linux/index.md]: Yes I mean remove the paintext password. We don't want a user that installs the tech preview, and later updates his docker and still have a password stored in plaintext (or base64 it's the same) left on the host.
HOwever, if it wasn't clear, I'm strongly against storing the password in plain text or base64. Just do the pass integration and we are done. The risk isn't worth the reward imho.

File: desktop/linux/index.md (hunk 15 above)
Comment (fredericdalleau) [reply desktop/linux/index.md:30]: You know like me that we don't like to reboot our linux machines 😄 
There are two parts : the system service, and the user access to the services.
So for the systemd service, I don't know if systemd is able to handle the new groups as the user is added into them or if the user need to be member of the group before the session service starts. But in that case, a logout/login cycle should be sufficient.
And for the user access to the services `su $USER` should do the trick.
But I agree I haven't tested it

**Review by usha-mandya — DISMISSED** [review]
LGTM

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**